core/assets/bundles/bundle_ja.properties

```
@@ -935,17 +1001,24 @@ mode.custom = カスタムルール
 
 rules.infiniteresources = 資源の無限化
 rules.reactorexplosions = リアクターの爆発
+rules.coreincinerates = コアの燃焼による破壊
```

CR: コアに入り切らなくなった分を焼却するかどうかの設定項目なはずです。
```suggestion
rules.coreincinerates = 余剰アイテムの焼却
```